api/controllers/topics-controllers.js

```diff
@@ -0,0 +1,6 @@
+const { fetchTopics } = require("../models/topics-models");
+
+exports.getTopics = async (req, res, next) => {
+  const topics = await fetchTopics();
+  res.status(200).send({ topics: topics.rows });
```

Comment:
topics.rows should have been returned in your model - the controller shouldn't do any data manipulation at all :)